core/math/plane.h

```diff
@@ -99,25 +100,18 @@ bool Plane::has_point(const Vector3 &p_point, real_t _epsilon) const {
 	return (dist <= _epsilon);
 }
 
-Plane::Plane(const Vector3 &p_normal, real_t p_d) {
-
-	normal = p_normal;
-	d = p_d;
+Plane::Plane(const Vector3 &p_normal, real_t p_d)
+	: normal(p_normal),
+	  d(p_d) {
 }
 
-Plane::Plane(const Vector3 &p_point, const Vector3 &p_normal) {
-
-	normal = p_normal;
-	d = p_normal.dot(p_point);
+Plane::Plane(const Vector3 &p_point, const Vector3 &p_normal)
+	: normal(p_normal),
+	  d(p_normal.dot(p_point)) {
 }
 
-Plane::Plane(const Vector3 &p_point1, const Vector3 &p_point2, const Vector3 &p_point3, ClockDirection p_dir) {
-
-	if (p_dir == CLOCKWISE)
-		normal = (p_point1 - p_point3).cross(p_point1 - p_point2);
-	else
-		normal = (p_point1 - p_point2).cross(p_point1 - p_point3);
-
+Plane::Plane(const Vector3 &p_point1, const Vector3 &p_point2, const Vector3 &p_point3, ClockDirection p_dir)
+	: normal((p_dir == CLOCKWISE) ? (p_point1 - p_point3).cross(p_point1 - p_point2) : (p_point1 - p_point2).cross(p_point1 - p_point3)) {
```

Comment:
That makes it less readable IMO, especially since the normal is then normalized in the constructor.